Core/Src/dashboard.c

```diff
@@ -351,7 +351,7 @@ void UpdateCockpitLed(uint32_t delay_100us) {
         //Control of other leds
         LED_RGB_setColor(LED_RGB1, LED1.R, LED1.G, LED1.B);
         LED_RGB_setColor(LED_RGB2, LED2.R, LED2.G, LED2.B);
-        LED_RGB_setColor(LED_RGB3, LED3.R, LED3.G, LED3.B);
+        LED_RGB_setColor(LED_RGB3, LED3.R, LED3.G, LED3.B);*/
```

Comment:
```suggestion
        LED_RGB_setColor(LED_RGB3, LED3.R, LED3.G, LED3.B);
 #endif
```
Never use comments (more in particular this ones `/* ... */` to block sections of code).
Reason: the `/*..*/` style of comments in C is very broken, look what happens in this case:
```c
// code that we want to comment out: start
......
#define SOME_VARIABLE (5U) /*!< Docs about variable */
functionCall();
.....
// code that we want to comment out: end
````
```c
/*
......
#define SOME_VARIABLE (5U) /*!< Docs about variable */
functionCall();
.....
*/
```
You can see `functionCall()` remains wrongfully active because on how `/* ... */` comments work. Therefore don't use comments to block code